tests/pkg_tinyvcdiff/fakemtd.h

```diff
@@ -65,7 +65,8 @@ typedef struct {
 #define FAKE_MTD_INIT { .mtd = { .driver = &fake_mtd_driver, \
                                  .sector_count = FAKE_MTD_SECTOR_COUNT, \
                                  .pages_per_sector = FAKE_MTD_PAGES_PER_SECTOR, \
-                                 .page_size = FAKE_MTD_PAGE_SIZE } }
+                                 .page_size = FAKE_MTD_PAGE_SIZE, \
+                                 .write_size = 1 } }
```

Comment:
Maybe setting this to 4 (`FAKE_MTD_WRITE_SIZE`) or would be more interesting

Reply:
As the write granularity is not enforced by the fake_mtd_driver, I wouldn't expect anything interesting.